docs/agile-with-CEA/activities.rst

```diff
@@ -2,37 +2,37 @@ Activities
 ===========
 
 This is a guide of Activities carried out during the development of CEA. This activities are part of our concept of operation
-udner responsability of our the Scrum Master
+under responsibilities of our the Scrum Master
 
 Planning event
 ---------------
 
-Duration: 1 - 4 hours.
-Frequency: After every block of work (sprint) (max of 1 month)
-Scope: Define what user stories should go in the next block of work (sprint). These stories are attached to a new time-bounded milestone
+* Duration: 1 - 4 hours.
+* Frequency: After every block of work (sprint) (max of 1 month)
+* Scope: Define what user stories should go in the next block of work (sprint). These stories are attached to a new time-bounded milestone
 Attendees: Scrum Master, Product Owner, Product Sponsor, and Development Team.
 
 Daily stand-up
 --------------
 
-Duration: 15 min.
-Frequency: daily
-Scope: Provide early support and mentoring
+* Duration: 15 min.
+* Frequency: daily
+* Scope: Provide early support and mentoring
 Attendees: Scrum Master and Development Team.
 
 Review event
 -------------
 
-Duration: 1 - 4 hours.
-Frequency: at the end of the sprint
-Scope: Present a realtime DEMO and receive feedback of the Product Sponsor.
+* Duration: 1 - 4 hours.
+* Frequency: at the end of the sprint
+* Scope: Present a realtime DEMO and receive feedback of the Product Sponsor.
 Attendees: Scrum Master, Product Owner, Product Sponsor, and Development Team.
 
 Retrospective
 -------------
 
-Duration: 1 hour.
-Frequency: After every review event
-Scope: Discuss what was wrong and set next steps to follow.
+* Duration: 1 hour.
+* Frequency: After every review event
```

Comment:
Perhaps this is called by the PO or SM on a less frequent basis when the team needs to talk about how good they are working and practices. Time to blow off some steam too.

Reply:
i think it is still good to have a defined structure and frequency.
and i think there are always things to discuss in retropect!